doc/source/text.rst

```diff
@@ -311,7 +311,7 @@ All one-dimensional list-likes can be arbitrarily combined in a list-like contai
 
```

Comment:
Another point of @TomAugspurger in #20347, responding to the line
```All one-dimensional list-likes can be arbitrarily combined in a list-like container (including iterators, ``dict``-views, etc.):```
(which is just above this line pointer):

@TomAugspurger 
> question: what happens with a dict? Do we use the keys, or does it transform to a series and get aligned?

Response 1:
> I was talking about `d.keys()` or `d.values()`. For passing a `dict` directly, currently the keys would get read (as that's what `x in d` would return).

Response 2:
> I could of course add another safety that maps `d` to `d.values()`.
However, I tend to think that this could be left to the python-skills of the user as well -- a dictionary is not "list-like" from the POV of normal python usage (whereas its keys and values are, see `d = dict(zip(keys, values))`).

Reply:
a dict is treated consistently here how we handle list-likes else where, we effectively call ``list`` on it